content/posts/about_inlining.md

```diff
@@ -9,11 +9,11 @@ tags = [
 
 ## Introduction
 
-For me, everytime I encounter the stituation and document myself about (chỗ này hơi bị bế quất, đọc lại vài lần rồi nói cho tao ý của mày nhé), I still end up with the same conclusions:
+For me, every time I encounter the `inline` keyword in codebase and document myself about it, I still end up with the same conclusions:
 
 > Inlining is often to be misused in development, ending up in counter-productive and risky situations.
```

Comment:
[Refactor] "to be" here is not neccessary, as you have put "is" after subject. 
[Suggestion (Optional)] The other part after colon can be written as: "which results in". Just a suggestion in case you want to write other blogs.